eunkyung/controllers/postController.js

```diff
@@ -0,0 +1,39 @@
+const postService = require('../services/postService')
+
+
+const posting = async (req, res) => {
+    const { title, content, postImageUrl, userId } = req.body
+    await postService.posting(title, content, postImageUrl, userId)
+    return res.status(201).json({ message: 'POSTING_SUCCESS' })
+}
+
+const userPost = async (req, res) => {
```

Comment:
함수명 수정해주세요!
`특정 유저가 작성한 게시글을 조회한다.`
```suggestion
const getPostByUserId = async (req, res) => {
```